ckan/public/base/javascript/modules/confirm-action.js

```diff
@@ -13,6 +13,17 @@ this.ckan.module('confirm-action', function (jQuery) {
        */
       content: '',
 
+      /* By default confirm-action creates a new form and submit it
+       * But you can use closest to el form by setting data-module-with-data=true
+       *
+       *     <a href="..."
+       *        data-module="confirm-action"
+       *        data-module-with-data=true>
+       *     {{ _('Save') }}
+       *     </a>
+       */
+      withData: '',
```

Comment:
if you are using default values, prefer `false` because this value is treated as flag